helm/chart/maesh/Chart.yaml

```
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: maesh
-version: 2.0.0
+version: 2.0.1
```

The Helm chart version must be updated in the Release PR (the Helm chart is not deployed or released on each commit like it is done for Traefik).

```suggestion
version: 2.0.0
```